app/controllers/dish_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DishController < ApplicationController
+  def show
+    @dish = Dish.find(params[:id])
+  end
+
+  def update
+    @ingredient = Ingredient.find(params[:ingredient_id])
```

Comment:
For this implementation, a more RESTful option would be to direct the form to a POST to a joins controller (could call it DishIngredients controller and direct it to a create action). In this case, you are creating an association between 2 existing resources rather than updating the attributes of a resource.